### ISSUE TITLE
`concurrent` api for all accounts, not just one

### ISSUE THREAD

**Issue by katherine-black** [cloudigrade/cloudigrade#666]
In GitLab by @infinitewarp on Mar 24, 2020, 18:30

## Summary

As the tally service, I want to ask cloudigrade for `concurrent` data for *all* known accounts in one call so that I don't have to iterate many calls, once per account.

This concern was raised when the tally service implemented its initial integration with cloudigrade. They expressed displeasure in having to construct and make multiple calls to cloudigrade for all of the relevant accounts. They would prefer to make one request and get all the data at once.

## Acceptance Criteria

- [ ] Verify this new API only returns results to internal trusted callers

## Assumptions and Questions

- This probably should happen after or alongside https://gitlab.com/cloudigrade/cloudigrade/-/issues/660 (Large data performance test for our data layer) and https://gitlab.com/cloudigrade/cloudigrade/-/issues/662 (Large data performance test for AWS interactions)
- Should we make this an internal private API somehow by messing with the URL structure?
- Should we make this require some kind of trusted service account?
- Should this API behave differently from concurrent, perhaps always only return one day's worth of data? Or perhaps we should impose a limit on the number of days requested via pagination? @infinitewarp is just concerned at the volume of data we may be processing all at once.

**Comment by katherine-black** [comment]
In GitLab by @infinitewarp on Mar 25, 2020, 10:51

We discussed this issue as a team during backlog refinement. We decided that we will not implement this request now. It should be safer for all parties involved to handle data in smaller chunks like we do today. Consider, for example, of we have one large/misbehaving account; we would rather clients be able to get data from all other accounts and miss this one than have one account spoil the whole bunch.

We will gladly reopen discussion in the future if needed. For now, we understand that developers on tally have a satisfactory implementation that iterates calls for all accounts even though that may not have been ideal initially from their perspective.
